packages/react-router/__tests__/setup.ts

```diff
@@ -45,3 +45,9 @@ if (!globalThis.File) {
   const { File } = require("undici");
   globalThis.File = File;
 }
+
+if (!globalThis.crypto?.subtle) {
```

Comment:
Not sure why, but Jest provides `crypto` but not `crypto.subtle`.

Reply:
Turns out it's because the tests were running in jsdom by default which polyfills `crypto` but in a non-secure context, so `crypto.subtle` provided. By flipping the failing tests to run in the Node env rather than jsdom, the `crypto` no longer exists in Node 18.